modules/users/client/components/ProfileViewBasics.js

```diff
@@ -4,7 +4,7 @@ import { withTranslation } from 'react-i18next';
 import { Trans } from 'react-i18next';
 import PropTypes from 'prop-types';
 import * as languages from '@/config/languages/languages';
-import { hasConnectedAdditionalSocialAccounts, isWarmshowersId, socialAccountLink } from './utils/networks';
```

Comment:
There was an existing pattern for `modules/**/client/utils/**`

https://github.com/Trustroots/trustroots/blob/70ebf948764773832dad6287e4bf15f077093bce/.eslintrc.js#L171